ParabolInc/parabol
# feat(metrics): Add Task Estimate Set event

# Description

Fixes #7116, based on #7067 
## Demo

<img width="388" alt="Screen Shot 2022-08-25 at 5 02 54 PM" src="https://user-images.githubusercontent.com/1879975/186789521-fdafe4bd-f9cd-4dba-8151-6569aa02fa8d.png">

<img width="431" alt="Screen Shot 2022-08-25 at 5 03 12 PM" src="https://user-images.githubusercontent.com/1879975/186789549-1ce4c12b-9ba6-4e30-8baf-1819b97e797e.png">


## Testing scenarios

- [ ] Success task estimate push
  - In the estimate phase of a poker meeting, try to push a score to Jira, with valid dimension setting
  - Verify the properties in Segment

- [ ] Failed task estimate push
  - In the estimate phase of a poker meeting, try to push a score to Jira, with invalid dimension setting (e.g., issues within Jira screens)
  - Verify the properties in Segment

## Final checklist
